examples/cra-ts-kitchen-sink/src/stories/docgen-tests/types/prop-types.js

```diff
@@ -140,6 +149,9 @@ PropTypesProps.propTypes = {
   obj: PropTypes.object, // eslint-disable-line react/forbid-prop-types
   symbol: PropTypes.symbol,
   node: PropTypes.node,
+  useCustomPropType: customPropType,
```

Comment:
i think you accidentally committed more than you meant?

Reply:
No, well I added a few more test cases to the kitche sink

Reply:
I can remote it

Reply:
Oh sorry I misunderstood the code. Nevermind!!